test/test-myEach.js

```diff
@@ -20,13 +20,28 @@ describe('myEach', function() {
     testArr = [0, 1, 100, 1000];
   });
 
-  it("takes a function as the second argument and calls that function (callback)", function testCallback() {
+  it("accepts an array as the first argument", function testFirstArgument() {
+    expect(true).to.equal(true); // this is just a freebie
```

Comment:
I know it goes against every tenant of good test design, but I think it will give students a better "road map" as they look at the tests.